R/rmd_format.R

```diff
@@ -53,64 +54,63 @@ attach_email <- function(message, preview = TRUE) {
         )
 
       writeLines(html, html_file)
-      browseURL(html_file)
+      utils::browseURL(html_file)
 
       # This sleep is necessary because knitting usually happens in a separate
       # process, and when that process terminates the temp file will be deleted
       Sys.sleep(5)
     }
   }
 
-  rmarkdown::output_metadata$set(rsc_email_body_html = message$html_str)
-  rmarkdown::output_metadata$set(rsc_email_images = message$images)
+  rmarkdown::output_metadata$set(rsc_email_body_html = email$html_str)
+  rmarkdown::output_metadata$set(rsc_email_images = email$images)
 
   invisible()
 }
 
-#' R Markdown output format for Blastula emails
+#' R Markdown output format for Blastula email messages
 #'
 #' @export
-blastula_email <- function(toc = FALSE, toc_depth = 3, toc_float = FALSE,
-  number_sections = FALSE, section_divs = TRUE, fig_width = 7,
-  fig_height = 5, fig_retina = 2, fig_caption = TRUE, dev = "png",
-  df_print = "default", code_folding = c("none", "show", "hide"),
-  code_download = FALSE, smart = TRUE, self_contained = TRUE,
-  theme = "default", highlight = "default", mathjax = NULL,
-  template = "blastula", extra_dependencies = NULL, css = NULL,
-  includes = NULL, keep_md = FALSE, lib_dir = NULL,
-  md_extensions = NULL, pandoc_args = NULL, ...) {
+blastula_email <- function(section_divs = TRUE,
+                           number_sections = FALSE,
+                           fig_caption = TRUE,
+                           smart = TRUE,
+                           dev = "png",
+                           self_contained = TRUE,
+                           template = "blastula",
```

Comment:
Add `fig_width = 5.35` (same for `fig_height` and `fig_retina` and `toc`, `toc_depth`, and `toc_float`).

Reply:
Now done.